### PR TITLE
parse JSON arrays and objects

### PR DESCRIPTION
**Pull request by paawaaa**
Objects like

```
{
  id: 0,
  name: 'string',
  arrayString: ['string', 'string', 'string'],
  arrayNumber: [0, 0, 0]
}
```

are now correctly converting to JSON type structure to put into DB.

### CODE REVIEW

File: lib/convert.js [hunk 1]
```diff
@@ -87,11 +95,17 @@ var convertParametersInner = function(change, convertString, param) {
           convertString = convertString.replace(reg, tempParamKey);
         } else {
           if (change == '#') {
-            tempParamKey = tempParamKey.toString().replace(/"/g, '\\\"');
-            tempParamKey = tempParamKey.replace(/'/g, '\\\'');
-            
-            convertString = convertString.replace(reg, "'" + tempParamKey + "'");
-          } else if (change == '$') {
+            if (change == '#') {
```
Comment (OldBlackJoe) [inline lib/convert.js]: I think there is no need use 'if (change == '#') {' twice.

Comment (OldBlackJoe) [inline lib/convert.js]: I think there is no need to use "if (change == '#') {" twice. this condition is used line 97 already.

**Comment by paawaaa** [comment]
Of course you are right! This is all because of the rush. Thank. Corrected.

File: lib/convert.js (hunk 1 above)
Comment (paawaaa) [reply lib/convert.js]: Of course you are right! This is all because of the rush. Thank. Corrected.

Comment (paawaaa) [reply lib/convert.js]: And I slightly corrected the indentation.

**Comment by paawaaa** [comment]
It would be nice if you update your [NPM package](https://www.npmjs.com/package/mybatis-mapper) to the latest version.

Something like:

```bash
$ npm version patch
$ npm publish
$ git push
```

**Pull request merged**